gips/scripts/export.py

```diff
@@ -30,9 +32,113 @@
 from gips.inventory import DataInventory, ProjectInventory
 from gips.inventory import orm
 
+# from backports import tempfile
+import tempfile
+import boto3
+import zipfile
+import shutil
+
+
+def get_s3_shppath(s3path, tmpdir):
```

Comment:
This is great.  Maybe we can move it to gips.utils and have `open_vector` use it automagically to make all of gips cli s3 enabled.